tests/returntypes/core/test_new_return_types_qnode.py

```diff
@@ -988,14 +1067,16 @@ def circuit(x):
     ]
 
     @pytest.mark.parametrize("device", devices)
-    @pytest.mark.parametrize("op1,wires1,op2,wires2", multi_probs_data)
+    @pytest.mark.parametrize("op1,wires1,op2,wires2", multi_probs_data[4:])
```

Comment:
Same as above, might be better to just have two lists.

Reply:
We could also have an additional `qubit_devices` lists to parameterize over for tests like these.